roles/ara_api/defaults/main.yaml

```
@@ -199,6 +202,9 @@ ara_api_database_port: null
 # ARA_DATABASE_CONN_MAX_AGE - Django's CONN_MAX_AGE database setting
 ara_api_database_conn_max_age: 0
 
+# ARA_DATABASE_OPTIONS - Django's OPTIONS database setting
+ara_api_database_options: null
```

The jobs are failing and I think it's unhappy with this being none -- can you try an empty dict instead ?

```
[...]
  File "/home/zuul/.ara-tests/virtualenv/lib64/python3.6/site-packages/django/db/backends/mysql/base.py", line 211, in get_connection_params
    options = settings_dict['OPTIONS'].copy()
AttributeError: 'NoneType' object has no attribute 'copy'
```

```suggestion
ara_api_database_options: {}
```